test/fixtures/schema.sql

```diff
@@ -970,6 +970,12 @@ create function test.ret_setof_integers() returns setof integer as $$
   values (1), (2), (3);
 $$ language sql;
 
+-- this function does not have named arguments and should be ignored
+-- if it's not ignored, it will break the test for the function before
+create function test.ret_setof_integers(int, int) returns integer AS $$
+  values (1);
+$$ language sql;
```

Comment:
Got it. How about adding #1638 to the same CHANGELOG entry:

https://github.com/PostgREST/postgrest/commit/e08bb3a197cbc03e5c623b9847651880955e45b8#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR28

Reply:
Absolutely!